### ISSUE TITLE
Community Name and Image

### ISSUE THREAD

**Issue by guilherme-miyake** [hubot-community/community-guidelines#1]
I have talked with Hubot's original creator Josh, and there might(?) be issues with using Hubot's name and logo in the future as both are GitHub owned.

As we are just getting started, there are many things to decide, this is truly not a critical issue and we might change our decision later.

/polls "We should use Hubot Name/Logo for now" "We should change it now"

**Comment by guilherme-miyake** [comment]
/polls "We should use Hubot Name/Logo for now" "We should change it now"

**Comment by mose** [comment]
Great move @guilherme-miyake you rock!

I suggest we brand this fork as hubot-ce for community edition, and that we get a new logo for it. I can propose one.

Note: `hubot-ce` is not used either on github nor on npmjs

**Comment by mose** [comment]
I don't think using hubot in the name would be a problem, provided that most hubot plugins do it for ages without concern.

**Comment by mose** [comment]
How about this?

![hubot-ce-logo1](https://user-images.githubusercontent.com/131461/106973978-dfa3c100-678e-11eb-82a8-1f37c00e1965.png)

**Comment by mose** [comment]
Here is the pitch that comes to explain the graphic choices on that visual design:

- hubot-ce is represented as a sort of smiley, a symbol that belongs to conversationnal online chatrooms.
- The yellow color is inherited from original hubot color but also agrees with the classic smiley standards.
- that smiley is a happy one, that's because all the connection it has. That face is not a known one, it is a specific thing with its own personality, not just the everyday smiley. With some imagination you could consider that the eyes are like `C` and the mouth could be a sort of `E` or a `H` but that would go a little far.
- it connectes to various dots, which represents the various adapters it can use to plug into various chat systems. But this multi-connection principle could also be intepreted with the intrinsec modularity of that bot, which can accept so many plugins. The length of the link that connects to the dots is long enough to make it clear that it is a link to a separate thing.

**Comment by guilherme-miyake** [comment]
I agree with both proposals, and we could start using them right now.

For the  logo, here are some things that could be improvements for future versions/revisions:
 - check logo color with a dark theme, it may be my impression but I think "darker" yellow might be better as the one is too bright
 - Change 'legs" distribution of nodes, the first impression I had was "hubot the overlord" on top of everything, I would proprose for a necktie (many sources, many functionalities) or human-like (which can be used as a "pet" for the project,) distribution.
 - I would personally like to play with the node sizes as well, not sure about this one
 
 Thanks for putting in the effort 👍🏾 , please add the image and its source file (if there is one) to the repo

**Comment by mose** [comment]
Here is an improved version:

![hubot-ce-logo2](https://user-images.githubusercontent.com/131461/108170927-be9b8280-7135-11eb-92dd-80341ab5ff79.png)

How does it look?

**Comment by mose** [comment]
Note: I changed the project avatar with that new image to see how it looks, and I like it, it scales well in small size too.
